DOCS-6375 Document relay log durability for semisynchronous replication (#844)
mariadb-corporation/mariadb-docs: server/ha-and-performance/standard-replication/replication-and-binary-log-system-variables.md

#### `relay_log_recovery`

* Description: If set to `1` (`0` is default), on startup the replica drops all [relay logs](../../server-management/server-monitoring-logs/binary-log/relay-log.md) that haven't yet been processed, and retrieve relay logs from the primary. Can be useful after the replica has crashed to prevent the processing of corrupt relay logs. relay\_log\_recovery should always be set together with [relay\_log\_purge](replication-and-binary-log-system-variables.md#relay_log_purge). Setting `relay-log-recovery=1` with `relay-log-purge=0` can cause the relay log to be read from files that were not purged, leading to data inconsistencies.

>   * This variable only has an effect on replicas that connect using binary log file and position coordinates. A replica that connects using [GTIDs](gtid.md) purges its relay logs every time the replication threads start, including after a restart, regardless of this setting.

Reviewer: ```suggestion
that connect using binary log file and position coordinates (i.e. `CHANGE MASTER TO MASTER_USE_GTID=NO`)
```
